src/Controller/CallbackApiController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace GoApptiv\FileManagement\Controller;
+
+use Illuminate\Http\Request;
+use GoApptiv\FileManagement\Services\FileManagement\FileManagementService;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Log;
+
+class CallbackApiController extends Controller
+{
+    /** @var FileManagementService */
+    private $fileManagementService;
+
+    /**
+     * Constructor.
+     *
+     * @param FileManagementService $fileManagementService
+     */
+    public function __construct(FileManagementService $fileManagementService)
+    {
+        $this->fileManagementService = $fileManagementService;
+    }
+
+    /**
+     * Update Variant Id by UUID
+     *
+     * @param Request $request
+     *
+     * @return bool
+     *
+     */
+    public function updateVariantIdByUuid(Request $request)
+    {
+        $uuid = $request['metadata']['uuid'];
+        $variantId = $request['metadata']['variantId'];
```

Comment:
uuid & variantId is required field? If it's required than please add it in custom request.